nixos/tests/keystone.nix

```diff
@@ -0,0 +1,79 @@
+{ system ? builtins.currentSystem }:
```

Comment:
do you want set yourself as maintainer for this test, so you get informed in case it fails?
```
  meta = with pkgs.stdenv.lib.maintainers; {                                                                                                       
    maintainers = [ ...];                                                                                                                             
  }; 
```